core/engine/src/builtins/temporal/duration/mod.rs

```diff
@@ -794,6 +795,12 @@ impl Duration {
             .with_message("not yet implemented.")
             .into())
     }
+
+    pub(crate) fn value_of(_this: &JsValue, _: &[JsValue], _: &mut Context) -> JsResult<JsValue> {
+        Err(JsNativeError::typ()
+            .with_message("valueOf not implemented for Temporal objects. See 'compare', 'equals', or `toString`")
```

Comment:
Hmm, I'm a bit negative about the "not implemented" phrasing because it seems like we haven't implemented it yet. Maybe something like "invalid call to `valueOf` for Temporal object".